### PR TITLE
NEP-152 notification issues

### PR DESCRIPTION
**Pull request by dev-siris**
@syuraj  sir yo commit review gardinu

### CODE REVIEW

**Comment by syuraj** [comment]
@dev-siris can you address the `DeepScan` issues first?
Also, please don't wait for me to approve, you can merge and test in qa, I can comment later on.

File: src/jobs/NotificationTrigger/__tests__/NotificationTrigger.integration.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const NotificationTrigger = require('../index')
 
-jest.setTimeout(20000)
+jest.setTimeout(200000)
```
Comment (syuraj) [inline src/jobs/NotificationTrigger/__tests__/NotificationTrigger.integration.js]: @dev-siris Why are you increasing timeout here? Tests should be completed in few seconds. If it takes more than few seconds, then, it is the problem with the test.

**Comment by syuraj** [comment]
Also, can you write a summary of the changes in PR title/description?

File: src/jobs/NotificationTrigger/notificationHelper.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const { NotificationDbService } = require('../../db-service')
+const { getStartEndTime } = require('./notificationTime')
```
Comment (syuraj) [inline src/jobs/NotificationTrigger/notificationHelper.js]: day start and end time  will be different for different user.

Comment (dev-siris) [reply src/jobs/NotificationTrigger/notificationHelper.js]: @syuraj  sir okay

File: src/jobs/NotificationTrigger/__tests__/NotificationTrigger.integration.js (hunk 1 above)
Comment (dev-siris) [reply src/jobs/NotificationTrigger/__tests__/NotificationTrigger.integration.js]: @syuraj  sir mero pc ma time badi lagi rako xa..

Comment (syuraj) [reply src/jobs/NotificationTrigger/__tests__/NotificationTrigger.integration.js]: @dev-siris it is something different issue, test shouldn't have even take 20 seconds

**Pull request merged**